sdk/identity/Azure.Identity/src/DateTimeHelpers.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Identity
+{
+    internal static partial class DateTimeHelpers
```

Comment:
`DateTimeHelpers` is required to match implementation of `System.Diagnostics.Activity` that requires precise time value.